src/containers/MapPage/MapPage.js

```diff
@@ -76,36 +77,51 @@ class MapsScreen extends Component {
 
   ShowDialog(museum, locName) {
     const locale = this.props.locale.toLocaleUpperCase();
+    const theme = this.props.theme;
+
     this.setState({
       showDialog : true,
       dialog: (
       <PopupDialog
-        dialogTitle={<DialogTitle title = {museum.name[locale]}/>}
+        dialogStyle={{backgroundColor: colors.BASE[theme],
+          height: 220,
+          borderTopWidth: 1,
+          borderBottomWidth: 1,
+          borderTopRightRadius: 10,
+          borderTopLeftRadius: 10,
+          borderColor: colors.MAIN,
+        }}
+        dialogTitle={<View style={{backgroundColor: colors.BASE[theme], borderBottomWidth: 0.5, borderColor: colors.MAIN}}>
+          <Text style={{color: colors.TEXT[theme], textAlign: 'center', fontSize: 20, fontFamily: fonts.EACH}}>
+            {museum.name[locale]}
+          </Text>
+        </View>}
         ref={(popupDialog) => { this.refDialog = popupDialog;}}
         dialogAnimation={new SlideAnimation({
           slideFrom: 'bottom',
         })}
         actions={[
           <DialogButton
+            textStyle = {{color: colors.MAIN, borderColor: colors.MAIN, fontSize: 30, fontFamily: fonts.MURRAY}}
             key = {id++}
-            text="More..."
+            text="More"
             onPress={()=>{
               this.props.navigation.navigate('MuseumItem', {data: museum, page: "Maps"});
-
               this.refDialog.dismiss();
             }}
           />
         ]}
       >
         <View>
-          <Text>
+          <Text style = {{color: colors.TEXT[theme], textAlign: 'center', marginTop: 20}}>
             <Image
-              align={"top"}
-              style={{width: 200, height: 200}}
+              style={{width: 230, height: 230}}
```

Comment:
Define the magic variables: 230

Reply:
I've changed it so now it correctly works in ios and android